<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="6" passed="4" failed="1" skipped="1">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-09-03T15:19:14 IST" name="Suite" finished-at="2023-09-03T15:19:36 IST" duration-ms="22805">
    <groups>
    </groups>
    <test started-at="2023-09-03T15:19:14 IST" name="Test" finished-at="2023-09-03T15:19:36 IST" duration-ms="22805">
      <class name="com.amazon.testcase.E2E_Flow">
        <test-method is-config="true" signature="reportGeneration()[pri:0, instance:com.amazon.testcase.E2E_Flow@2b71e916]" started-at="2023-09-03T15:19:14 IST" name="reportGeneration" finished-at="2023-09-03T15:19:14 IST" duration-ms="176" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- reportGeneration -->
        <test-method is-config="true" signature="intializeBrowser()[pri:0, instance:com.amazon.testcase.E2E_Flow@2b71e916]" started-at="2023-09-03T15:19:14 IST" name="intializeBrowser" finished-at="2023-09-03T15:19:17 IST" duration-ms="3492" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- intializeBrowser -->
        <test-method signature="OneTest()[pri:1, instance:com.amazon.testcase.E2E_Flow@2b71e916]" started-at="2023-09-03T15:19:17 IST" name="OneTest" finished-at="2023-09-03T15:19:18 IST" duration-ms="416" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- OneTest -->
        <test-method signature="TwoTest()[pri:2, instance:com.amazon.testcase.E2E_Flow@2b71e916]" started-at="2023-09-03T15:19:18 IST" name="TwoTest" finished-at="2023-09-03T15:19:20 IST" duration-ms="2076" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TwoTest -->
        <test-method signature="ThreeTest()[pri:3, instance:com.amazon.testcase.E2E_Flow@2b71e916]" started-at="2023-09-03T15:19:20 IST" name="ThreeTest" finished-at="2023-09-03T15:19:25 IST" duration-ms="5348" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ThreeTest -->
        <test-method signature="FourTest()[pri:4, instance:com.amazon.testcase.E2E_Flow@2b71e916]" started-at="2023-09-03T15:19:26 IST" name="FourTest" finished-at="2023-09-03T15:19:36 IST" duration-ms="10377" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- FourTest -->
        <test-method signature="FiveTest()[pri:0, instance:com.amazon.testcase.E2E_Flow@2b71e916]" started-at="2023-09-03T15:19:36 IST" name="FiveTest" finished-at="2023-09-03T15:19:36 IST" depends-on-methods="com.amazon.testcase.E2E_Flow.FourTest" duration-ms="108" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [false] but found [true]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertFalse(Assert.java:79)
at org.testng.Assert.assertFalse(Assert.java:89)
at com.amazon.testcase.E2E_Flow.FiveTest(E2E_Flow.java:47)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- FiveTest -->
        <test-method signature="SixTest()[pri:0, instance:com.amazon.testcase.E2E_Flow@2b71e916]" started-at="2023-09-03T15:19:36 IST" name="SixTest" finished-at="2023-09-03T15:19:36 IST" depends-on-methods="com.amazon.testcase.E2E_Flow.FiveTest" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method com.amazon.testcase.E2E_Flow.SixTest() on instance com.amazon.testcase.E2E_Flow@2b71e916 depends on not successfully finished methods 
[com.amazon.testcase.E2E_Flow.FiveTest() on instance com.amazon.testcase.E2E_Flow@2b71e916]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method com.amazon.testcase.E2E_Flow.SixTest() on instance com.amazon.testcase.E2E_Flow@2b71e916 depends on not successfully finished methods 
[com.amazon.testcase.E2E_Flow.FiveTest() on instance com.amazon.testcase.E2E_Flow@2b71e916]
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:156)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SixTest -->
        <test-method is-config="true" signature="closingBrowser()[pri:0, instance:com.amazon.testcase.E2E_Flow@2b71e916]" started-at="2023-09-03T15:19:36 IST" name="closingBrowser" finished-at="2023-09-03T15:19:36 IST" duration-ms="58" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closingBrowser -->
        <test-method is-config="true" signature="closingReport()[pri:0, instance:com.amazon.testcase.E2E_Flow@2b71e916]" started-at="2023-09-03T15:19:37 IST" name="closingReport" finished-at="2023-09-03T15:19:37 IST" duration-ms="228" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closingReport -->
      </class> <!-- com.amazon.testcase.E2E_Flow -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
